one/the_number_is_five.rb

```diff
@@ -1,8 +1,8 @@
-number = 5
-
-if(number == 5)
-  puts "the number is five"
-else
-  puts "the number is NOT five"
+def number_is_five(number)
+  if(number == 5)
```

Comment:
Please make this change.
number.eql?(5) ? 'the number is five' : the number is NOT five'

Reply:
Done.